glide.yaml

```diff
@@ -77,13 +77,15 @@ import:
   repo:    https://github.com/openshift/containers-image.git
   version: openshift-3.8
 # pod - sjenning - from branch openshift-3.10-runc-595bea0
+# origin-3.11-runc-871ba2e
 - package: github.com/opencontainers/runc
   repo:    https://github.com/openshift/opencontainers-runc.git
-  version: da78c1f1ff59f174e7307e9ff76e06308aefac4e
+  version: 3de28e9df02ff2da57bf6f504dd78c182f0e188e
 # pod - sjenning
+# origin-3.11-cadvisor-v0.30.1 MISSING PATCHES
```

Comment:
@sjenning **patches do not apply**.  This is going to backlevel you.  You'll need to update the fork.

Reply:
fork has been updated https://github.com/openshift/google-cadvisor/pull/3 @runcom